jablib/src/main/java/org/jabref/logic/quality/consistency/BibliographyConsistencyCheck.java

```diff
@@ -14,16 +15,22 @@
 import org.jabref.logic.bibtex.comparator.BibEntryByCitationKeyComparator;
 import org.jabref.logic.bibtex.comparator.BibEntryByFieldsComparator;
 import org.jabref.logic.bibtex.comparator.FieldComparatorStack;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.database.BibDatabaseMode;
 import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.BibEntryType;
 import org.jabref.model.entry.field.Field;
 import org.jabref.model.entry.field.SpecialField;
 import org.jabref.model.entry.field.StandardField;
 import org.jabref.model.entry.field.UserSpecificCommentField;
+import org.jabref.model.entry.types.BiblatexEntryTypeDefinitions;
+import org.jabref.model.entry.types.BibtexEntryTypeDefinitions;
 import org.jabref.model.entry.types.EntryType;
 
 public class BibliographyConsistencyCheck {
 
     private static final Set<Field> EXPLICITLY_EXCLUDED_FIELDS = Set.of(
+                            StandardField.KEY,
```

Comment:
I am not sure about this one, but I thinkj, it is OK to leave this in. 

Also include all from "JabRef-specific fields" (org.jabref.model.entry.field.StandardField#CITATIONCOUNT etc)

Also includde `org.jabref.model.entry.field.InternalField#KEY_FIELD`.